src/core/dxf/qgsdxfexport.h

```diff
@@ -385,10 +385,19 @@ class CORE_EXPORT QgsDxfExport
     void writeMText( const QString &layer, const QString &text, const QgsPoint &pt, double width, double angle, const QColor &color );
 
     /**
-     * Calculates a scaling factor to convert from map units to a specified symbol unit.
-     * The \a scale parameter indicates the scale denominator, e.g. 1000.0 for a 1:1000 map.
-     */
-    static double mapUnitScaleFactor( double scale, QgsUnitTypes::RenderUnit symbolUnits, QgsUnitTypes::DistanceUnit mapUnits );
+     * Returns scale factor for conversion to map units
+    * @param scaleDenominator the map scale denominator
```

Comment:
If they are written as `\param` they will be picked up correctly by sipify for the API generation.
And while touching this, would be nice to have them indented in a unified way.